third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -103,11 +103,7 @@ const TEST_CONFIGS: &[TestConfig] = &[
     TestConfig {
         name: "lambda",
         runner: |p| run(p, get_config_by_name("lambda")),
-        experiments: &[
-            (Experiment::OPTIMIZE, true),
-            (Experiment::FUNCTION_VALUES, true),
-            (Experiment::LAMBDA_LIFTING, true),
-        ],
+        experiments: &[(Experiment::FUNCTION_VALUES, true)],
```

Comment:
Should we add a TODO for testing with optimize=false and other configs tested above? Or perhaps the expectation is that eventually, this experiment will be on by default, and "lambda" will not be excluded in the above configs.